app/models/pawn.rb

```diff
@@ -6,4 +6,39 @@ def image
       '&#9823;'
     end
   end
+
+  def valid_move?(x, y)
+    dx = delta_x(x_position, x)
+    dy = delta_y(y_position, y)
+
+    return false if blocked?(x, y)
+
+    # pawn can't move horizontally
+    return false if dx != 0 && dy.zero?
+
+    # pawn moves vertically 1 space
+    return true if dx.zero? && dy == 1
+    return false if color == 'white' && y_position < y
```

Comment:
You can use an inline if statement (the one with a `?` and `:`) in order to make these two statements into one:

```ruby
return false if (color == 'white' ? y_position < y : y_position > y)
```

Or probably more clear:

```ruby
wrong_direction = color == 'white' ? y_position < y : y_position > y
return false if wrong_direction
```

But you actually already have a method to detect a backwards move.  👍  So, this could just be:

```ruby
return false if backwards_move?(y)
```